web/src/components/project/component/detail/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
Please try not to implement the actual logic in the `index.js`. Move the logic to other file and keep the index as just for exporting.

Reply:
👌